invenio_app_ils/config.py

```diff
@@ -658,6 +658,20 @@ def _(x):
             order=1
         )
     ),
+    eitems=dict(  # ItemSearch.Meta.index
```

Comment:
Is this `# ItemSearch.Meta.index` required or information that is used as an index to item search

Reply:
This is a reminder for us that the name should be same as the `Record.index`.